### ISSUE TITLE
Add version information to API

### ISSUE THREAD

**Issue by gavv** [roc-streaming/roc-java#57]
C library provides version info: https://roc-streaming.org/toolkit/docs/api/reference.html#roc-version

It would be nice to add this feature to the java library too. We need to provide two version actually: version of the native library (libroc), and version of the bindings (which may be slightly different).

This issue requires updating both JNI and java parts.

**Comment by saurabhyadav1985** [comment]
@gavv Kindly assign this issue to me in case it hasn't been claimed yet.

**Comment by gavv** [comment]
Sure, thanks!

**Comment by gavv** [comment]
@saurabhyadav1985 Hi, do you still have plans on this?

**Comment by ayushkumar105** [comment]
@gavv Can I work on this issue and [83](https://github.com/roc-streaming/roc-java/issues/83)?

**Comment by gavv** [comment]
Sure, thanks! 

Note that roc-java is not yet updated to the latest roc-toolkit, so you'll need to install roc-toolkit from 0.2.x tag rather than from master.

**Comment by gavv** [comment]
@ayushkumar105 Hi, still have plans on this?

**Comment by ayushkumar105** [comment]
@gavv Yes, although I will only have time during December. Currently my college semester is ongoing so I don't really have time.

**Comment by gavv** [comment]
Unassigning so that someone could pick this up. @ayushkumar105 Feel free to ping me to be re-assigned if you'll have time for the issue.
